### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**
Initial PR

### CODE REVIEW

File: mayhem/FuzzedDataProvider.h [hunk 1]
```diff
@@ -0,0 +1,397 @@
+//===- FuzzedDataProvider.h - Utility header for fuzz targets ---*- C++ -* ===//
```
Comment (xansec) [inline mayhem/FuzzedDataProvider.h]: Please don't include FDP.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: mayhem/FuzzedDataProvider.h (hunk 1 above)
Comment (capuanob) [reply mayhem/FuzzedDataProvider.h]: @xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**